var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -39,11 +39,15 @@ class Mesa(MesonPackage):
 
     # Internal options
     variant('llvm', default=True, description="Enable LLVM.")
-    variant('swr', default='auto',
-            values=('auto', 'none', 'avx', 'avx2', 'knl', 'skx'),
+    _SWR_AUTO_VALUE = 'auto'
+    _SWR_ENABLED_VALUES = [_SWR_AUTO_VALUE, 'avx', 'avx2', 'knl', 'skx']
+    _SWR_DISABLED_VALUE = 'none'
+    variant('swr', default=_SWR_AUTO_VALUE,
+            values=(_SWR_DISABLED_VALUE,) + tuple(_SWR_ENABLED_VALUES),
```

Comment:
```suggestion
    _SWR_ENABLED_VALUES = (_SWR_AUTO_VALUE, 'avx', 'avx2', 'knl', 'skx')
    _SWR_DISABLED_VALUES = ('none',)
    variant('swr', default=_SWR_AUTO_VALUE,
            values=_SWR_DISABLED_VALUES + _SWR_ENABLED_VALUES,
```

I think using tuples here directly makes this easier to read. Do you agree?

Reply:
Yes!